doc/releases/release-notes-3.2.rst

```diff
@@ -358,6 +358,7 @@ Drivers and Sensors
   * Atmel eefc: Fix support for Cortex-M4 variants.
   * Added flash driver for Renesas Smartbond platform
   * STM32: Added OSPI NOR-flash driver. Supports STM32H7 and STM32U5. Supports DMA.
+  * NRF: Added CONFIG_SOC_FLASH_NRF_TIMEOUT_MULTIPLIER to allow tweaking the timeout of flash operations.
```

Comment:
This needs to go into `release-notes-3.3.rst` now.

Reply:
oops sorry will rebase